### ISSUE TITLE
Waistline will probably be removed from Google Play Store

### ISSUE THREAD

**Issue by davidhealey** [davidhealey/waistline#773]
Google is asking me to verify myself by providing them with ID documents, residence, phone number, and email address. I don't want to give them this information so they are probably going to remove my Google Play store developer account along with Waistline.

![image](https://github.com/davidhealey/waistline/assets/14021202/cccf0f5f-0a61-4490-b943-d1dde0c2b17f)

I've reserved a date of December 2024, so we should be good until then at least.

**Comment by Nitrousoxide** [comment]
If you are going to remove it, it is probably a good idea to push an update that gives a popup once to alert users to switch to f-droid and provide them a link, so they don't silently stop getting updates.  I can't imagine that most users will be checking the github.

**Comment by davidhealey** [comment]
Good idea

**Comment by Yushi5058** [comment]
Hello,
You can try to do an APK for the app. There are open source apps that I use who use this approach. An APK in an official website of the app.
Hope it helps,
Yushi

**Comment by davidhealey** [comment]
> Hello, You can try to do an APK for the app. There are open source apps that I use who use this approach. An APK in an official website of the app. Hope it helps, Yushi

An APK is included with every release here on github

**Comment by Yushi5058** [comment]
> > Hello, You can try to do an APK for the app. There are open source apps that I use who use this approach. An APK in an official website of the app. Hope it helps, Yushi
> 
> An APK is included with every release here on github

Then no need for an alternative to play store after removal when you can have the apk, all good .

**Comment by DJPapa66** [comment]
Hello.
F-Droid is a very good alternative. It is my first choice when looking for apps.

**Comment by fs-99** [comment]
> Around 1.5k

if fdroid provides stats as well, how many users do you have there? I've only found this app on fdroid, so I'm assuming more than on gplay

**Comment by davidhealey** [comment]
> if fdroid provides stats as well, how many users do you have there? I've only found this app on fdroid, so I'm assuming more than on gplay

I don't believe f-droid provides that info. At least I don't see it anywhere.
